frontend/benefit/handler/src/components/applicationReview/salaryBenefitCalculatorView/SalaryBenefitCalculatorView.tsx

```diff
@@ -372,10 +386,181 @@ const SalaryBenefitCalculatorView: React.FC<
           invalid={!!getErrorMessage(fields.endDate.name)}
           aria-invalid={!!getErrorMessage(fields.endDate.name)}
           errorText={getErrorMessage(fields.endDate.name)}
-          style={{ paddingRight: `${theme.spacing.xs}` }}
+          style={{ paddingRight: `${theme.spacing.s}` }}
         />
       </$GridCell>
 
+      {!isManualCalculator && (
+        <>
+          <$GridCell $colStart={1} $colSpan={11}>
+            <$CalculatorText
+              style={{ marginTop: theme.spacing.m, fontWeight: 'bold' }}
```

Comment:
$CalculatorText is styled component so I would prefer to put styles inside the component parameterized like 
`<<$CalculatorText bold />` or prefer `css` atribute instead of `style .

Reply:
This is now fixed